src/components/forms/form-field-styles.ts

```diff
@@ -5,10 +5,16 @@ import { INPUT_BORDER_RADIUS, INPUT_PADDING } from '../../utils/constants';
 export const FormFieldContainer = styled.div`
   display: flex;
   flex-direction: column;
+`;
+
+export const Input = styled.input`
+  padding: ${INPUT_PADDING};
+  border-radius: ${INPUT_BORDER_RADIUS};
+  border: ${INPUT_BORDER};
+`;
 
-  input {
-    padding: ${INPUT_PADDING};
-    border-radius: ${INPUT_BORDER_RADIUS};
-    border: ${INPUT_BORDER};
-  }
+export const InputSelect = styled.select`
```

Comment:
Aqui esse componente e o de cima possuem o mesmo estilo e o código está duplicado, o estiled component nos dá uma opção de criar um css que pode ser reaproveitado, então vc poderia fazer algo do tipo:
```suggestion
export const BaseInput = css` 
 padding: ${INPUT_PADDING};
  border-radius: ${INPUT_BORDER_RADIUS};
  border: ${INPUT_BORDER};
`;
```
e ai dizer que seus outros estilos possuem esse objt css, assim:
```suggestion
export const Input = styled.input`
${BaseInput}
`;
```